WalletWasabi.Fluent.Desktop/Program.cs

```diff
@@ -43,6 +40,29 @@ public static int Main(string[] args)
 
 		Logger.LogDebug($"Wasabi was started with these argument(s): {(args.Any() ? string.Join(" ", args) : "none") }.");
 
+		(UiConfig uiConfig, Config config) = LoadOrCreateConfigs(dataDir);
+		using SingleInstanceChecker singleInstanceChecker = new(config.Network);
+
+		try
+		{
+			singleInstanceChecker.EnsureSingleOrThrowAsync().GetAwaiter().GetResult();
```

Comment:
It seems to me that starting the client twice is not reported by the CrashReporter. The app just silently fails to start, is that correct? 
The SingleInstanceChecker not just checks but also signal to the other instance - if that was successful we do not need to report anything but if not then we should show the crashreport window - otherwise the user thinks nothing happened after the start.